generator/uscore/metadata_extractor.rb

```diff
@@ -180,7 +181,7 @@ def add_must_support_elements(profile_definition, sequence)
               sequence[:must_supports] <<
                 {
                   type: 'element',
-                  path: path.gsub('[x]', type['code'].slice(0).capitalize + type['code'].slice(1..-1))
+                  path: path.gsub('[x]', capitalize_first_letter(type['code']))
```

Comment:
```suggestion
                  path: path.gsub('[x]', type['code'].capitalize)
```

Reply:
Can't do this because it makes the rest of string lowercase. dateTime becomes Datetime.